qctrlopencontrols/version.py

```diff
@@ -18,4 +18,4 @@
 ===========================
 """
 
-__version__ = '1.1.1'
+__version__ = '1.2.1'
```

Comment:
Just saw the email, should this be 1.2.0?

Reply:
Urgh, since I messed up it has to become 1.2.1